service.subtitles.rvm.addic7ed/addic7ed/webclient.py

```diff
@@ -40,7 +40,7 @@ def _open_url(self, url, params, referer):
         logger.debug('Opening URL: {0}'.format(url))
         self._session.headers['Referer'] = referer
         try:
-            response = self._session.get(url, params=params)
+            response = self._session.get(url, params=params, verify=False)
```

Comment:
Why disable the SSL verification? That does not seem safe?

Reply:
As it turned out, they use Let's Encrypt certificate and fail to renew it from time to time: https://forum.kodi.tv/showthread.php?tid=222672&pid=3041515#pid3041515